codes.json

```
@@ -743,21 +743,22 @@
   ,
   {
     "title" : "Osiris",
-    "subtitle" : "TBD/",
-    "description" : "TBD/",
-    "application": [],
-    "compute": [],
-    "method": [],
+    "subtitle" : "an object-oriented parallel 3D PIC code for modeling laser and particle beam-plasma interactions",
+    "description" : "An object-oriented, high performance fully explicit EM-PIC code",
+    "application": ["plasma based accelerators", "radiation sources"],
+    "compute": ["laptop/desktop", "linux clusters", "HPC centers"],
+    "method": ["PIC"],
     "phenomena": [],
     "homepage" : "http://picksc.idre.ucla.edu/software/software-production-codes/osiris",
-    "institutions" : ["TBD/"],
-    "institution_link" : ["TBD/"],
-    "repository" : "TBD/",
-    "issue_tracker" : "TBD/",
+    "institutions" : ["UCLA", "IST/Lisbon"],
+    "institution_link" : ["http://www.ucla.edu/"],
+    "repository" : "https://github.com/GoLP-IST/osiris",
+    "issue_tracker" : "https://github.com/GoLP-IST/osiris/issues",
     "platform": [],
```

@tsung1029 the supported platforms are Linux/Unix and macOS (i.e. no Windows)?
```suggestion
    "platform": ["linux", "macos"],
```